backoff_test.go

```diff
@@ -0,0 +1,11 @@
+package grpc
+
+import "testing"
+
+func TestBackoffConfigDefaults(t *testing.T) {
+	b := BackoffConfig{}
+	b.setDefaults()
+	if b != *DefaultBackoffConfig {
```

Comment:
You cannot have constant structs in Go. We can make this a concrete value to get the same effect. In fact, making this a value was [discussed in the original PR](https://github.com/grpc/grpc-go/pull/601#discussion_r57981280) and we went with the pointer type.

I'll make this change in a separate commit in case this isn't what you want.